java/Cherepnin_Oleh/src/com/zoolatech/lecture7/tasks/_2/SyncCalculator.java

```diff
@@ -0,0 +1,25 @@
+package com.zoolatech.lecture7.tasks._2;
+
+public class SyncCalculator implements Calculator {
+    private volatile int value;
```

Comment:
you don't need to define a variable as `volatile` if all operation that work with it are protected with the synchronized block - it provides a same guarantees (memory values will be synchronized)